src/components/enterprise-user-subsidy/data/hooks/hooks.js

```diff
@@ -244,3 +245,28 @@ export function useCustomerAgreementData(enterpriseId) {
 
   return [customerAgreement, isLoading];
 }
+
+export function useSubsidyLearnerCredit(enterpriseId, userID) {
+  const [learnerCreditSubsidies, setLearnerCreditSubsidies] = useState();
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
[nit/suggestion] FWIW, I might recommend looking into using `@tanstack/react-query` to make this API call so we don't have to worry about setting up your own custom `isLoading` state variables, etc. (overall less boilerplate when working with async data fetching + other nice benefits like automated query retries in case the request fails).

See https://openedx.atlassian.net/wiki/spaces/AC/pages/3791290378/tanstack+react-query+An+overview#useQuery for some documentation on `useQuery`.

Note: just wanted to add `useQuery` as an option to explore; I defer to your judgment if you'd like to use it or not (IMO, it makes sense to rely on `@tanstack/react-query` for any net-new API integrations, and eventually we may migrate existing API calls over to `@tanstack/react-query` to simplify the implementations and mitigate some performance concerns).

Reply:
Done